### PR TITLE
MPT-44 TestDiscardProtocol is unstable

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: mpt/core/pom.xml [hunk 1]
```diff
@@ -61,6 +56,21 @@
             <groupId>org.awaitility</groupId>
             <artifactId>awaitility</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>compile</scope>
```
Comment (mbaechler) [inline mpt/core/pom.xml]: why do you need that scope ? AFAIK, we don't set the scope in parents

File: mpt/core/src/test/java/org/apache/james/mpt/DiscardProtocolTest.java [hunk 2]
```diff
@@ -77,15 +79,17 @@ void tearDown() {
...
+
         String output = record.complete();
+
         assertThat(output).isEqualTo(INPUT);
     }
 
     private void input() throws IOException {
         Writer out = new OutputStreamWriter(socket.getOutputStream());
         out.append(INPUT);
+        out.flush();
```
Comment (mbaechler) [inline mpt/core/src/test/java/org/apache/james/mpt/DiscardProtocolTest.java]: from the javadoc :
```
close() Closes the stream, flushing it first.
```

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**